### PR TITLE
Add the recipe of Agua de Valencia

### PR DESCRIPTION
**Pull request by alejandroereyesb**

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, seems like `aquapanela.jpg` was changed? 

Also, can you rename your image file to match the json filename?
```
aguadevalencia.jpg -> agua-de-valencia.jpg`
```

File: src/recipes/agua-de-valencia.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            "measure": "spoon",
+            "ingredient": "sugar"
+        }
+    ],
+    "directions": [
+        "Start by making fresh-squeezed orange juice (if you can find Valencia oranges, all the better). It may be a bit labor intensive, but trust me, you will taste the difference! Fresh-squeezed orange juice has a much lighter, fresher taste that will make your Agua de Valencia super refreshing. In Spain, many grocery stores have orange juice machines where you can simply press a button to get a liter of Valencia’s famous orange juice. Taste your orange juice and take note of its sweetness. If its on the bitter side, you may need to add some sugar.",
+        "Pour all the liquid ingredients in your pitcher. Mix together and taste. If you used a bitter cava or type of champagne, this will also affect how much sugar needs to be added. Agua de Valencia is not meant to be a super sugary drink, but it isn’t meant to be bitter either!",
+        "Grab a couple of large wine glasses and pour your fresh Spanish cocktail over ice. Chop a few orange slices and add them as a garnish. Clink your glasses, say “¡Salud!” and enjoy!"
+    ],
+    "image": "aguadevalencia.jpg",
```
Comment (alfg) [inline src/recipes/agua-de-valencia.json:37]:
```suggestion
    "image": "agua-de-valencia.jpg",
```
Use same name convention as recipe file.

Comment (alejandroereyesb) [reply src/recipes/agua-de-valencia.json]: Sure! I fixed the problem with name convention and added aquapanela original image again. Thanks!

**Comment by alfg** [comment]
Thanks for fixing!

**Pull request merged**